packages/flutter/lib/src/loader/README.md

```diff
@@ -0,0 +1,3 @@
+This directory contains a small Flutter application that is run while
```

Comment:
I don't think we should put this in packages/flutter/lib/src/, since it's not part of the framework.

Some possible options:
- bin/loader/
- packages/loader/
- packages/flutter_tools/loader/

I'm not sure what the best option is. In the three examples above it could have a pubspec.yaml to help the analyzer.